examples/pid/specification.yml

```diff
@@ -0,0 +1,42 @@
+user_claims:
+  iss: https://pid_issuer.memberstate.eu
+  iat: 1541493724
+  type: PersonIdentificationData
+  binding:
+  - !sd
+    type: CryptographicBinding
+    cnf:
+      jwk:
+        kty: EC
+        crv: P-256
+        x: TCAER19Zvu3OHF4j4W4vfSVoHIP1ILilDls7vCeGemc
+        y: ZxjiWWbZMQGHVWKVQ4hbSIirsVfuecCE6t4jT9F2HZQ
+  - !sd
+    type: BiometricBinding
+    template: "..."
+  !sd given_name: Erika
+  !sd family_name: Mustermann
+  !sd nationalities: [!sd "DE"]
+  !sd birth_family_name: Schmidt
+  !sd birthdate: '1973-01-01'
+  !sd place_of_birth: Regensburg
```

Comment:
[ARF PID Rulebook v.1.0.0] specifies ‘Place of Birth’ as an optional data element in a PID attribute, definfing  the following data elements related to this PID attribute:

∙ birth_place (optional)
∙ birth_country (optional)
∙ birth_state (optional)
∙ birth_city (optional)

or a JSON object like this

````
"place_of_birth":{
    "region":"MI",
    "locality":"F205",
    "country": ...,
    "state": ...,
    "city": ...,
}
````